apps/ima/views/declaration.html

```diff
@@ -1,9 +1,9 @@
+<title>{{#t}}pages.declaration.header{{/t}} – GOV.UK</title>
 {{<partials-page}} {{$page-content}}
```

Comment:
put `{{$page-content}}` on the next line

Reply:
{{$page-content}}    moved to next line